fabtests/functional/multi_mr.c

```diff
@@ -269,7 +269,6 @@ static int run_test(void)
 
 	ret = mr_key_test();
 
-	ft_sync();
```

Comment:
Can we leave ft_sync and `ft_finalize` separate? You just need to add a missing `ft_sync` in pingpong right?

Reply:
Sure, will do that